rpcs3/Emu/RSX/GL/OpenGL.h

```diff
@@ -6,7 +6,7 @@
 #ifdef _WIN32
 #include <Windows.h>
 #include "GL/gl.h"
-#include "GL/glext.h"
+#include "externals/GL/glext.h"
```

Comment:
Is it possible to include "externals" directory so that #include <GL/glext.h> works ?
I prefer the <> syntax for non local includes.